src/widgets/category/category-colors.js

```diff
@@ -1,5 +1,8 @@
 var _ = require('underscore');
-var colors = ['#2CA095', '#E5811B', '#4A4DBA', '#AD2BAD', '#559030', '#E1C221']; // Demo colors
+var colorScales = [['#2CA095', '#E5811B', '#4A4DBA', '#AD2BAD', '#559030', 'rgb(166,206,227)', 'rgb(31,120,180)', 'rgb(178,223,138)', 'rgb(51,160,44)', 'rgb(251,154,153)', 'rgb(27,158,119)', 'rgb(217,95,2)', 'rgb(117,112,179)', 'rgb(231,41,138)', 'rgb(102,166,30)', 'rgb(230,171,2)', '#E1C221'],
```

Comment:
@fdansv @javisantana @andrewxhill we'll have some new qualitative palettes ready in the next day. If you have any suggestions on the best way to provide those, please let me know. Right now we have the quantitative palettes (sequential not diverging _yet_) here: https://github.com/CartoDB/CartoColor/blob/master/cartocolor.js